corehq/messaging/scheduling/views.py

```diff
@@ -229,10 +236,15 @@ def message_form(self):
             (instance.recipient_type, instance.recipient_id)
             for instance in schedule_instances
         ]
+        ret = []
+        if recipients:
+            for doc_type, doc_id in recipients:
+                user = CommCareUser.wrap(CommCareUser.get_db().get(doc_id))
```

Comment:
i realize we're only supporting users for this pr, but we'll need to make this conditional based on doc type eventually